synchronization/src/androidTestCyface/java/de/cyface/synchronization/UploadProgressTest.java

```diff
@@ -1,7 +1,9 @@
 package de.cyface.synchronization;
 
 import static de.cyface.synchronization.CyfaceConnectionStatusListener.SYNC_PERCENTAGE;
+import static de.cyface.synchronization.SharedConstants.DEVICE_IDENTIFIER_KEY;
 import static de.cyface.synchronization.TestUtils.*;
```

Comment:
no wildcard imports

Reply:
I searched and replaced all wildcard imports in the code